build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -7,7 +7,14 @@ import { Flags } from "@oclif/core";
 import chalk from "chalk";
 
 import { BaseCommand } from "../../base";
-import { Repository, createPullRequest, pullRequestExists, pullRequestInfo } from "../../lib";
+import { Repository, createPullRequest, pullRequestExists } from "../../lib";
+import { getCommitInfo } from "../../lib/github";
```

Comment:
`getCommitInfo` can be exported https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/src/lib/index.ts#L34
```suggestion
import { Repository, createPullRequest, pullRequestExists, getCommitInfo } from "../../lib";
```

Reply:
Good catch.